src/fuzz.zig

```diff
@@ -19,6 +22,13 @@ pub const FuzzFilter = enum {
     gossip_table,
 };
 
+fn spawnMetrics(gpa_allocator: std.mem.Allocator) !std.Thread {
+    const metrics_port: u16 = config.current.metrics_port;
+    logger.default_logger.infof("metrics port: {d}", .{metrics_port});
+    const registry = globalRegistry();
+    return std.Thread.spawn(.{}, servePrometheus, .{ gpa_allocator, registry, metrics_port });
+}
```

Comment:
can we move this into the prometheus file and call it from both here and `cmd.zig` (shouldnt need `spawnMetrics`) - seems like code duplication rn